CHANGELOG.md

```diff
@@ -54,6 +54,8 @@ For experimental package changes, see the [experimental CHANGELOG](experimental/
   [#3359](https://github.com/open-telemetry/opentelemetry-js/pull/3359) @dyladan
 * fix(resources): strict OTEL_RESOURCE_ATTRIBUTES baggage octet decoding
   [#3341](https://github.com/open-telemetry/opentelemetry-js/pull/3341) @legendecas
+* fix(version): updated webpack version for selenium test issue
```

Comment:
Usually the component goes into the parenthesis. :slightly_smiling_face:
```suggestion
* fix(selenium-tests): updated webpack version for selenium test issue
```

Also this should be moved up into the "unreleased" section :slightly_smiling_face: